src/backend/distributed/executor/citus_custom_scan.c

```diff
@@ -318,3 +318,25 @@ CitusReScan(CustomScanState *node)
 							   "parameters are currently unsupported")));
 	}
 }
+
+
+/*
+ * ScanStateGetTupleDescriptor returns the tuple descriptor for the given
+ * scan state.
+ */
+extern TupleDesc
```

Comment:
Minor: We typically don't have `extern` qualifier on the source files.

Reply:
Fixed. Thanks.